docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 version: "3.2"
 services:
-  dev: # 
+  dev:
```

Comment:
I'm not going to open a separate PR to remove one errant character, so that's that.